### DISCUSSION TITLE
Welcome to BORIS Discussions!

### DISCUSSION THREAD

**Discussion by olivierfriard** [olivierfriard/BORIS#345] [General]
## 👋 Welcome to the BORIS discussions space!

Please
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by weejosdf** [comment]
Hello:  A quick question.  We have had some students running boris, and unfortunately when they set up the parameters they did not include any focal animals.  They do have a variable titled "animal id".   Is it possible to enter the proper animal id into the subjects column in the observations list after the videos have been scored?

**Reply by olivierfriard** [reply]
Yes, you can edit the recorded events (one- by one or in a group). 
Select the events > right-click to open the context menu > Edit selected events

**Reply by weejosdf** [reply]
Thank you for the prompt reply.  As I dig into this it appears to be somewhat more involved.  The ethogram the students are using contains an independent variable labeledl "Animal ID", but nothing for subjects, which means there is nothing to enter when editing the recorded events.  I am guessing the next step is to modify the ethogram to include all the subjects now listed under animal ID, then edit all the events?

And wait, there's more!  Various video segments (using this subject-less-ethogram) have been scored on videos stored on multiple different hard drives.  It appears around 4 drives, with multiple drive numbers (same drive, plugged into different computer ports at different times.  sigh).  Is it possible to move all the videos and observation lists to a single drive without having to re-record the data?  

thanks for the help.....I am hoping this is the worst of the issues!
Sean

**Comment by JoliVL** [comment]
Hello! I was wondering if BORIS allows for the offset time to be in milliseconds? 
I would like to sync two videos of the same behaviour from two different angles. When checking the offset time necessary for them to play at the same time, Adobe Premiere shows me that I should put it at 3,16 seconds, but BORIS seems to only accept 3,1 or 3,2. 


Thank you very much!

**Comment by olivierfriard** [comment]
Hi,

What version of BORIS are you using?

The offset should be set in the observation window like you can see in the
attached screenshot

o

On Tue, May 25, 2021 at 11:46 AM JoliVL ***@***.***> wrote:

> Hello! I was wondering if BORIS allows for the offset time to be in
> milliseconds?
> I would like to sync two videos of the same behaviour from two different
> angles. When checking the offset time necessary for them to play at the
> same time, Adobe Premiere shows me that I should put it at 3,16 seconds,
> but BORIS seems to only accept 3,1 or 3,2.
> Worst case scenario, I could mute video number 2 as the behaviour is
> somewhat the same, just a little bit delayed. However, I also don't see
> this option. Is this possible?
>
> Thank you very much!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/olivierfriard/BORIS/discussions/345#discussioncomment-780284>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABR2AYQOSSZINQOGYV4DLVTTPNWYHANCNFSM4UUFV6MA>
> .
>


-- 
University of Torino
Dept of Life Sciences and Systems Biology
Via dell'Accademia Albertina, 13 10123 TORINO (Italy)
tel: +39 011 6704542
http://penelope.unito.it/friard/
http://orcid.org/0000-0002-0374-9872

**Reply by JoliVL** [reply]
Hi, thank you for your quick reply! The screenshot is not visible as an attachment, so I have attached what I did myself in this comment.
I am using version 7.10.5, and this is what I write in the offset - so I can write 5.16, but it seems that there is no difference in the offset time from writing 5.1. Is it correct that BORIS cannot deal with those numbers?
Thank you!
![BorisOffset](https://user-images.githubusercontent.com/64142082/119671952-7aa5d100-be3a-11eb-926c-3cf88a2d640e.JPG)

**Comment by charlottebodie** [comment]
Hi Oliver,

I am just wondering whether it is possible to group individual species together in BORIS? I.e, combining key events by species so that I can plot events (by species and not as individuals) after the videos have been coded. 

Thank you!

Charlotte

**Comment by rvtmsw** [comment]
Hello Oliver,
The revisions have still used the old logo on the icon (logo_eye.ico) lately.
Especially the Windows portable version.

Cheers, Riva.
